src/docs/cookbook/gestures/dismissible.md

```diff
@@ -106,28 +111,25 @@ the indicator is a red background.
 To add the indicator,
 provide a `background` parameter to the `Dismissible`.
 
-<!-- skip -->
-```dart
-Dismissible(
-  // Show a red background as the item is swiped away.
-  background: Container(color: Colors.red),
-  key: Key(item),
-  onDismissed: (direction) {
-    setState(() {
-      items.removeAt(index);
-    });
-
-    ScaffoldMessenger
-        .of(context)
-        .showSnackBar(SnackBar(content: Text("$item dismissed")));
-  },
-  child: ListTile(title: Text('$item')),
-);
+<?code-excerpt "lib/{step2,main}.dart (Dismissible)"?>
+```diff
+--- lib/step2.dart (Dismissible)
++++ lib/main.dart (Dismissible)
+@@ -16,6 +16,8 @@
+       ScaffoldMessenger.of(context)
+           .showSnackBar(SnackBar(content: Text("$item dismissed")));
+     },
++    // Show a red background as the item is swiped away.
++    background: Container(color: Colors.red),
+     child: ListTile(title: Text('$item')),
+   );
+ },
```

Comment:
I've used the `diff` here, because it shows better where the change is (adding the background to the item)